### DISCUSSION TITLE
Kotlin ImageMagick bindings

### DISCUSSION THREAD

**Discussion by MolotovCherry** [ImageMagick/ImageMagick#4553] [General] [answered]
Hello,

I'm the original author of the Imagemagick Android port (the one listed on the https://imagemagick.org/ homepage), and I wanted to announce that I've completed some Kotlin bindings for Imagemagick. I was wondering if I could get that listed on your [official bindings page](https://imagemagick.org/script/develop.php) on your website. 

You can find my Kotlin bindings project [here](https://github.com/cherryleafroad/kmagick)

**Comment by dlemstra** [comment, marked as answer]
The source of our website is also hosted on GitHub. You can open a pull request in this repository: https://github.com/ImageMagick/Website to add your project to that list.

**Reply by MolotovCherry** [reply]
Thanks for the pointer 😃
